### ISSUE TITLE
Feature: Support for Materials Symbols

### ISSUE THREAD

**Issue by Poiuytrezay1** [ng-icons/ng-icons#63]
**Feature description**
A while back, Google published [Material Symbols](https://material.io/blog/introducing-symbols), a better version of material icons giving designers & developers more control over the icons' looks. They are also packaged within a variable font.

Can they be added to ng-icons too?

The list of icons is available here: https://fonts.google.com/icons (some very useful ones like "left panel close" are not in the default material icons font).

**Comment by ashley-hunter** [comment]
Thank you for the suggestion! Currently all our icons are SVG based so this would be quite different and not be able to share the existing component or providers.

If icon fonts were going to be supported, ideally we would create a generic component that could handle any icon font, not exclusively variable icon fonts.

Another consideration is that the font variation settings are not standard across icon fonts, so that would be something that would be need to be considered how we could support all possible variations, ideally with inputs that could be type safe.

I would like to add this support for this, although it will need some thought.

**Comment by Nxtivision** [comment]
Hello @ashley-hunter ,

Any update on this? 😄 We would like to use your lib with the use of Material Symbols

**Comment by ashley-hunter** [comment]
Hi, yes, I have been working on it for a while now. I have just made the 26.3.0 release which comes with experimental support for it. It's not currently documented on the site, but you can find documentation on how to use it here:

https://github.com/ng-icons/ng-icons#variable-icon-fonts

This is an early preview, so there may be a few issues - let me know and I will resolve them. The API is still in flux, so if you have any feedback, things you find difficult, or something missing let me know and we can make the necessary changes before I mark it as "stable".

Thanks

**Comment by Nxtivision** [comment]
Hi @ashley-hunter 

I tried this experimental feature and it looks very promising ! Ease of use, I succeeded to replace many svg icon using Material Symbols by this `ng-glyph`.

However, the doc is quite misleading. I had to check inside the generated CSS to see how to use the different properties:

```
provideNgGlyphsConfig({
  size: '20px',
  color: undefined,
  weight: 400,
  grade: 0,
  fill: false,
})
```

`size` should be a `px` unit, or `rem`, `em`,... Whatever you want.
Also, using TailwindCSS, the `color: currentColor` override the value, so I had to explicitly set it to `undefined`

These few changes should be enough.

Thank you for your time on this feature ! 👏
